packages/loot-core/src/platform/server/connection/index.d.ts

```diff
@@ -1,7 +1,7 @@
 import type { ServerEvents } from '../../../types/server-events';
 
 export function init(
-  channel: Window | string,
+  channel: Window | string | int,
```

Comment:
TS doesn't have int/float types. Also, is the `string` case used anymore?

```suggestion
  channel: Window | string | number,
```